File: wealth.js
function two(increase){
    let max=0;
    for(i=0;i<increase.length;i++){
    let one=0;
    for(let j=0;j<increase[i].length;j++){
    one+=increase[i][j];
    }
    if(one>max){
    max=one;
    }
    }
    return max;
    }
    let arrys=[[6, 645, 797, 201], [214, 564, 854, 201], [265, 564, 564, 2345]]
    let fixit=two(arrys)
    console.log(fixit)

File: wealth.java
public class wealth{
    public static void main(String[] args){
       int[][] anu={{65,56,64,32},{56, 65, 23, 56}};
       int fine= siva(anu);
       System.out.println(fine);
    }
    static int siva(int[][] one){
       int max=0;
    for(int i=0; i<one.length;i++){
     int increase=0;
     for(int j=0;j<one[i].length;j++){
       increase+=one[i][j];
     }
    if(increase>max){
       max=increase;
    }
   }
   return max;
    }
   }